config/initializers/sorcery.rb

```diff
@@ -15,7 +13,7 @@
 
   config.github.key = ENV["OUATH_GITHUB_KEY"]
   config.github.secret = ENV["OUATH_GITHUB_SECRET"]
-  config.github.callback_url = "#{SERVER}/oauth/callback?provider=github"
+  config.github.callback_url = "#{ENV["APP_HOST"]}/oauth/callback?provider=github"
```

Comment:
Line is too long. [82/80]<br>Prefer single-quoted strings inside interpolations.